### PR TITLE
Format arg-name in ast.scm

### PR DESCRIPTION
**Pull request by xlxs4**

### CODE REVIEW

**Comment by Seelengrab** [comment]
The old indentation was lined up with the `if` in the other cases, no?

**Review by vtjnash — APPROVED** [review]

File: src/ast.scm [hunk 1]
```diff
@@ -329,8 +329,8 @@
         (else
          (case (car v)
            ((...)
-	    (arg-name (cadr v)) ;; to check for errors
-	    (decl-var (cadr v)))
+	        (arg-name (cadr v)) ;; to check for errors
+	        (decl-var (cadr v)))
```
Comment (JeffBezanson) [inline src/ast.scm:333]: I think the real problem is that there are tab characters on this line, and the other lines use spaces. So, just convert the tabs to spaces instead.

Comment (xlxs4) [reply src/ast.scm]: TIL `M-x untabify`

File: src/ast.scm [hunk 2]
```diff
@@ -226,13 +226,13 @@
...
                         "")
                     (string.rep "    " ilvl) "end"))
-	   ((do)
-	    (let ((call (cadr e))
-		  (args (cdr (cadr (caddr e))))
-		  (body (caddr (caddr e))))
-	      (deparse-block (string (deparse call) " do" (if (null? args) "" " ")
-				     (deparse-arglist args))
-			     (cdr body) ilvl)))
+       ((do)
```
Comment (vtjnash) [inline src/ast.scm:229]: It looks like this block moved to the wrong indent (I am not sure it was right before either, as seems it was not aligned with the (try) above but was aligned with the (struct) below it)

Comment (vtjnash) [reply src/ast.scm:229]: Looks right to me

**Pull request merged**